user_class/spec/user_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative '../user'
+require 'factories'
+
+RSpec.describe User do
+  subject(:users) { described_class.new }
+
+  let(:info_user) { attributes_for(:user) }
+
+  describe '.create' do
+    context 'when all arguments are passed' do
+      it 'add an user' do
+        expect { users.create(**info_user) }.to change(users, :count).by(1)
+      end
+
+      it 'does not add an user' do
```

Comment:
Enclose this within a new conetxt.
```suggestion
context "when the user id already exist" do
...
      it 'does not add an user' do
      ...
```